### ISSUE TITLE
Support newlines in inline tables

### ISSUE THREAD

**Issue by theoparis** [BurntSushi/toml#380]
I tried to use traefik, which uses this library for supporting toml config files.

Example traefik config that I used
```toml
certificatesResolvers.leresolver = {
	acme = {
		email = "theop@example.com",
		tlsChallenge = {},
	},
}

http.routers.main = {
	rule = "Host(`code.example.com`)",
	tls = {
		certResolver = "leresolver",
	},
}
```

The error from docker compose:
```
command traefik error: toml: line 1 (last key \"certificatesResolvers.leresolver\"): newlines not allowed within inline tables
```

Please let me know if this an issue with traefik specifically and not a parser issue.

See  https://github.com/toml-lang/toml/pull/904

**Comment by arp242** [comment]
That feature isn't in any tagged TOML release yet, so that's why this library doesn't support it.

That said, I did add support for this last week in #378. I also added the `BURNTSUSHI_TOML_110` environment variable to enable it in bd94408; you'll have to rebuild traefik to use the latest master of this library, and with that env var multiline inline tables (and other TOML 1.1 features) should work.

*But* as said none of this is in a tagged TOML release yet. Anything can still change in the TOML specification. Your TOML files that are valid now can break with future updates.
